### ISSUE TITLE
EasyCLA not able to fetch the commit author information from GitHub bots

### ISSUE THREAD

**Issue by jarias-lfx** [communitybridge/easycla#4441]
## Summary

Support has received several reports from EasyCLA not longer been able to fetch the commit author information from GitHub bots.

## Background

EasyCLA fetches the author information from the commit and validates the author agains EasyCLA database to conform the presence of an ECLA or ICLA. Currently this is failing for all GitHub bots and EasyCLA reports Missing ID on Commit

![image](https://github.com/user-attachments/assets/eccba6dd-a12e-4b41-bebf-b15bf5a6e718)

Affected PRs:

- https://github.com/cloudfoundry/korifi/pull/3478
- https://github.com/cloudfoundry/app-autoscaler-release/pull/3210
- https://github.com/finos/legend-studio/pull/3529
- https://github.com/finos/legend-studio/pull/3530
- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3642
- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3645
- https://github.com/cloudfoundry/uaa/pull/3050
- https://github.com/cloudfoundry/uaa/pull/3051
- https://github.com/webpack/webpack/pull/18770
- https://github.com/AcademySoftwareFoundation/openexr/pull/1824

## Expected behavior

EasyCLA should be able to fetch the author information from GitHub bots.

## Workaround

Please contact the project owners or maintainers to force merge the PR without EasyCLA for the GitHub bots if urgent, while EasyCLA investigates the issue.

**Comment by jarias-lfx** [comment]
It seems to be working again.

Note for [AlanGreene](https://github.com/AlanGreene): /easycla comment is not working in tektoncd, we have requested the owners to check of there is configuration that is blocking comment actions in the organization. Maybe closing/re-open the PR work.

**Comment by AlanGreene** [comment]
Thanks @jarias-lfx. I'll work with the other maintainers to see if we can unblock the `/easycla` command as there are a few dozen PRs affected and I'd rather not have them all re-trigger all of their CI jobs.

**Comment by afrittoli** [comment]
I think the problem is that EasyCLA is asking for new permissions that we have not granted yet:

<img width="855" alt="image" src="https://github.com/user-attachments/assets/f76c61f4-3985-4387-b600-74a3629b373e">

What does EasyCLA need read/write administration of the GitHub org for? 
We use peribolos to maintain some of org settings and I would like to avoid having conflicting changes from EasyCLA.

@vdemeester FYI

**Comment by vdemeester** [comment]
@afrittoli ah good point. I think I missed those "new permissions" request then 😓

**Comment by AlanGreene** [comment]
> What does EasyCLA need read/write administration of the GitHub org for?

@jarias-lfx is that something you can answer?

**Comment by nickmango** [comment]
@mlehotskylf will need your help in checking the easycla bot admin config settings

**Comment by mlehotskylf** [comment]
@afrittoli @vdemeester These are the original permissions that must be accepted when the EasyCLA GitHub app is first installed. Without them, EasyCLA may not function as intended. It’s important to note that EasyCLA isn’t requesting administrative write access to gain full control over your repository. Instead, it needs this level of permission because core features—such as setting up webhooks or enforcing mandatory checks—fall under GitHub’s “Administration” category. Since these capabilities are integral to EasyCLA’s operation, the app must request write access to administration.

For reference, here are all the permissions EasyCLA is requesting:

![Image](https://github.com/user-attachments/assets/f512c84d-b9f0-46d2-9fd5-ca1f8a631343)
![Image](https://github.com/user-attachments/assets/326bb23d-9e9b-41f1-8cd9-c68778cedaca)
![Image](https://github.com/user-attachments/assets/1231250b-1d1d-4c14-ad51-b47a9e7b2291)

CC: @jarias-lfx

**Comment by mlehotskylf** [comment]
Closing this ticket as resolved.
@afrittoli @vdemeester please feel free to open new ticket if you have any more questions.
Thank you!
